src/PlaylistContainer/PlaylistContainer.tsx

```diff
@@ -1,30 +1,40 @@
 import React from 'react';
 import SavedPlaylist from '../SavedPlaylists/SavedPlaylist';
 import GeneratedPlaylist from '../GeneratedPlaylists/GeneratedPlaylist';
-import CustomPlaylist from '../CustomPlaylists/CustomPlaylist';
+// import CustomPlaylist from '../CustomPlaylists/CustomPlaylist';
+
+import { CleanedAlbumTrack } from '../utils';
+
+interface IProps {
+	playlistType: string,
+	selectedGenre: string,
+	playlists: [CleanedAlbumTrack[]] | []
+}
 
 enum PlaylistTypes {
 	SavedPlaylist = 'saved-playlist',
 	GeneratedPlaylist = 'generated-playlist',
-	CustomPlaylist = 'custom-playlist',
+	// CustomPlaylist = 'custom-playlist',
 }
 
-const displayPlaylist = (playlistType: PlaylistTypes) => {
+const displayPlaylist = (playlistType: string, playlists: [CleanedAlbumTrack[]] | [], selectedGenre: string) => {
 	if (playlistType === PlaylistTypes.SavedPlaylist) {
 		return <SavedPlaylist />;
 	}
-	if (playlistType === PlaylistTypes.GeneratedPlaylist) {
-		return <GeneratedPlaylist />;
-	}
-	if (playlistType === PlaylistTypes.CustomPlaylist) {
-		return <CustomPlaylist />;
+	if (playlistType === PlaylistTypes.GeneratedPlaylist && playlists.length) {
```

Comment:
This conditional is a bit long, but it is required because of the way TypeScript deals with union types currently.